### DISCUSSION TITLE
Fixing the parameters in Webots simulation

### DISCUSSION THREAD

**Discussion by NarimanNiknejad** [bitcraze/discussions#1452] [Q&A]
Hi,

Based on https://github.com/bitcraze/crazyflie-simulation/issues/60, there are some inconsistencies in the parameters when compared to the actual system. I was wondering if you could help me figure out the actual values we need to put in the proto file to represent the actual Crazyflie. The reason for this question is that if we want to use the model-based control on the simulation, the values do not match and cause instability in the drone. 

Thanks is advance.

**Comment by ataffanel** [comment]
Kimberly here at Bitcraze that is specialized in this is currently on vacation. Hopefully someone else from the community can answer. You may also try to request the WeBots community.

My gut feeling though would be that the simulated Crazyflie model and simulator are not perfect and so different tuning are required. My understanding is that simulators like Webots are mostly used for higher-level behavioral simulation so are not well suited for tuning of developing model-based. But this is something you might want to double-check with the Webots community.

**Reply by NarimanNiknejad** [reply]
Thank you so much for your response. That is how I feel too. But it would be great to make them such that model-based controllers can be implemented for testing.
